ql/src/java/org/apache/hadoop/hive/ql/security/authorization/plugin/HivePrivilegeObject.java

```diff
@@ -152,24 +151,31 @@ public enum HivePrivObjectActionType {
   private String rowFilterExpression;
 
   public HivePrivilegeObject(HivePrivilegeObjectType type, String dbname, String objectName) {
```

Comment:
Shall we deprecate the old constructors?

Reply:
What does dbName mean for a `DFS_URL` or a `SERVICE_NAME`?

Reply:
My mistake, these constructors can be optimized.